interfaces/html.idl

```diff
@@ -1,30 +1,9 @@
-// HTML IDLs
-typedef (Int8Array or Uint8Array or Uint8ClampedArray or
-         Int16Array or Uint16Array or
-         Int32Array or Uint32Array or
-         Float32Array or Float64Array or
-         DataView) ArrayBufferView;
```

Comment:
This makes no difference, because ArrayBufferView is only used here in a function parameter, and idlharness can't observe the types of function parameters.  I don't see any way this is likely to change.  We should make WebIDL/interfaces.html and then it can be included by other callers that need it, but there's no reason to include it for the HTML IDL tests, and it doesn't need to block this IDL update.